srcDLL/DllMain.cpp

```diff
@@ -104,11 +110,22 @@ bool InstallDepPatch()
 }
 
 
+// Redirects Outpost2.ini to be accessed from the OPU directory.
+void RedirectIniFile()
+{
+	const auto iniPath = GetOutpost2IniPath();
+	Op2MemCopy(0x00547090, iniPath.length() + 1, iniPath.data());  // Overwrite gConfigFile.iniPath
+}
```

Comment:
The destination buffer is MAX_PATH-sized. While it could be bounds-checked, if we're trying a path longer than MAX_PATH, we're already going to explode due to OS limits. (Win10 did add large path support, but apps need to opt-in to it via the manifest and be coded to use them, and there's no real chance of doing that for OP2.)